packs/bestiary-effects/effect-invoke-rune.json

```diff
@@ -0,0 +1,156 @@
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 16
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Bestiary"
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.NPCAbility.RuneGiant.InvokeRune.RuneOfDestruction.Label",
+                        "value": "destruction"
+                    },
+                    {
+                        "label": "PF2E.NPCAbility.RuneGiant.InvokeRune.RuneOfFlames.Label",
+                        "value": "flames"
+                    },
+                    {
+                        "label": "PF2E.NPCAbility.RuneGiant.InvokeRune.RuneOfSmiting.Label",
+                        "value": "smiting"
+                    }
+                ],
+                "flag": "rune",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.NPCAbility.RuneGiant.InvokeRune.Prompt",
+                "rollOption": "invoke-rune:rune"
+            },
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.Weapon.Base.greatsword",
+                        "value": "greatsword"
+                    },
+                    {
+                        "label": "PF2E.Weapon.Base.longspear",
+                        "value": "longspear"
+                    },
+                    {
+                        "label": "PF2E.BattleForm.Attack.Rock",
+                        "value": "rock"
+                    }
+                ],
+                "flag": "weapon",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.Weapon",
+                "rollOption": "invoke-rune:weapon"
+            },
+            {
+                "key": "Note",
+                "outcome": [
+                    "criticalSuccess",
+                    "success"
+                ],
+                "predicate": [
+                    "invoke-rune:rune:destruction"
+                ],
+                "selector": "{item|flags.pf2e.rulesSelections.weapon}-attack",
+                "text": "PF2E.NPCAbility.RuneGiant.InvokeRune.RuneOfDestruction.Text",
+                "title": "PF2E.NPCAbility.RuneGiant.InvokeRune.RuneOfDestruction.Label",
+                "visibility": "owner"
+            },
+            {
+                "definition": [
+                    "item:slug:greatsword"
+                ],
+                "key": "AdjustStrike",
+                "mode": "add",
+                "predicate": [
+                    "invoke-rune:rune:destruction",
+                    "invoke-rune:weapon:greatsword"
+                ],
+                "property": "weapon-traits",
+                "value": "deadly-3d12"
+            },
```

Comment:
We can cut down the number of REs here by pairing the two weapons.

`["item:slug:{item|flags.pf2e.rulesSelections.weapon}", {or": [ "item:slug:greatsword", "item:slug:rock" ] } ]`